doc/radosgw/admin.rst

```diff
@@ -2,25 +2,23 @@
  Admin Guide
 =============
 
-Once you have your Ceph Object Storage service up and running, you may
-administer the service with user management, access controls, quotas 
-and usage tracking among other features.
-
+After the Ceph Object Storage service is up and running, it can be administered
+with user management, access controls, quotas, and usage tracking.
 
 User Management
 ===============
 
-Ceph Object Storage user management refers to users of the Ceph Object Storage
-service (i.e., not the Ceph Object Gateway as a user of the Ceph Storage
-Cluster). You must create a user, access key and secret to enable end users to
+Ceph Object Storage user management refers only to users of the Ceph Object
+Storage service and not to the Ceph Object Gateway as a user of the Ceph
+Storage Cluster. Create a user, access key and secret to enable end users to
 interact with Ceph Object Gateway services.
 
-There are two user types: 
+There are two types of user: 
 
-- **User:** The term 'user' reflects a user of the S3 interface.
+- **User:** The term "user" refers to  user of the S3 interface.
 
-- **Subuser:** The term 'subuser' reflects a user of the Swift interface. A subuser
-  is associated to a user .
+- **Subuser:** The term "subuser" refers to a user of the Swift interface. A
```

Comment:
S3 can have subusers too, though this isn't obvious.

Reply:
I'm making a note of this for myself, and I will return to this in the future when I have a better understanding of this information and a better idea of how to incorporate it into the rest of the information here.